bitnami/redis/Chart.yaml

```diff
@@ -24,4 +24,4 @@ maintainers:
 name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
-version: 16.8.9
+version: 16.8.10
```

Comment:
I think we shouldn't bump the chart version since the chart itself is not being modified at any point

Reply:
Yes, you are right.
This was just a trick to trigger the verification and check if the PR worked or not.
I should have added a comment on this but I thought that the title of the last commit was enough.
I am reverting that change.